bin/git-commands.js

```diff
@@ -64,7 +64,8 @@ async function commit(message) {
 }
 
 async function pushToRemote(remote, branchName) {
-  const stdout = await execAsPromise(`git push --set-upstream ${remote} ${branchName}`)
+  // const stdout = await execAsPromise(`git push --set-upstream ${remote} ${branchName}`)
```

Comment:
Accidentally left comented-out code?

Reply:
Yep, totally left this in there. I'll check the old way, since the problem I'd had was with the token.